From 472cd7862bd8c348aab23c6972a6e25bea729ac1 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Sat, 12 Jan 2019 00:06:01 +0200
Subject: [PATCH 1/1] Added Caffeine Popover to list of popovers which should
 get padding.

Also consolidated the list since there was a redundant frame.container padding setting for Places and User Indicator popovers.
---
 src/gtk/sass/common/_3rd-party.scss | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/src/gtk/sass/common/_3rd-party.scss b/src/gtk/sass/common/_3rd-party.scss
index c8f14a9..87ba699 100644
--- a/src/gtk/sass/common/_3rd-party.scss
+++ b/src/gtk/sass/common/_3rd-party.scss
@@ -372,15 +372,13 @@ window.budgie-popover:not(.csd) {
   }
 
   &.budgie-menu,
+  &.caffeine-popover,
+  &.night-light-indicator,
+  &.places-menu,
   &.user-menu,
   &.workspace-popover {
     > frame.container { padding: $popover_padding; }
   }
-
-  &.night-light-indicator,
-  &.places-menu {
-    > frame.container { padding: $popover_padding; }
-  }
 }
 
 // icon-tasklist popover styling
-- 
2.20.1

